doc/content/source/problem_operators/problem_operator.md

```diff
@@ -0,0 +1,9 @@
+# ProblemOperator
+
+## Summary
+
+`ProblemOperator` objects are `mfem::Operators` that are called inside `MFEMExecutioner` to solve a
+step of the FE problem and update the `mfem::BlockVector` storing the true degrees of freedom of all
+variables.
```

Comment:
Consider adding links to the relevant MFEM docs.

Reply:
Added